source/frontend/src/features/properties/map/shared/SidebarFooter.tsx

```diff
@@ -18,18 +20,31 @@ const SidebarFooter: React.FunctionComponent<ISidebarFooterProps> = ({
   onSave,
   onCancel,
   isOkDisabled,
+  isMissingFields,
 }) => {
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const saveFields = () => {
+    onSave();
+    if (isMissingFields) {
+      setErrorMessage('Please check the required fields.');
+    }
+  };
+
   return (
     <SidebarFooterBar className="justify-content-end mt-auto no-gutters">
       {(!showEdit || editMode) && (
         <>
+          <Col xs="auto" className="pr-3" style={{ paddingTop: '0.7rem', color: 'red' }}>
```

Comment:
this can likely be done with the className: "invalid-feedback"
Also in general the issue with inlining styles like this is that it makes the css very hard to override - the specificity is very high. Also inlined styles aren't reusable in the way that classnames or styled components are.

Reply:
I tried using that class but the message format and placement was a little off so instead I went with a styled component way of styling.